extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -62,7 +62,7 @@ const InstructionDetails = styled.div`
 `;
 
 const ResourceTitle = styled(Stack)`
-  margin: 10px 0;
+  margin: 5px 0;
```

Comment:
We should try to use multiples of 4.
```suggestion
  margin: 4px 0;
```